packages/kbn-test/src/legacy_es/es_test_config.js

```diff
@@ -6,10 +6,14 @@
  * Side Public License, v 1.
  */
 
+import { REPO_ROOT } from '@kbn/utils';
```

Comment:
I feel like we should just be importing `kibanaPackageJson` all over rather than using `REPO_ROOT` and `loadJsonFile` and `Path.join()` all over.

Reply:
Changed it!

Reply:
Do you think that is good to go?